yal/assessment_data/A2_locus_of_control.py

```diff
@@ -0,0 +1,110 @@
+# Assessment state naming convention
+# state_a[number]_[subsection]_q[question]_topic
+ASSESSMENT_QUESTIONS = {
+    # A2
+    "1": {
+        "start": "state_a2_1_q1_loc_of_ctrl",
+        "questions": {
+            "state_a2_1_q1_loc_of_ctrl": {
+                "type": "list",
+                "text": "\n".join(
+                    [
+                        "_*I'm my own boss.*_ 😎",
+                    ]
+                ),
+                "options": [
+                    "Not at all true",
+                    "A little true",
+                    "Kind of true",
+                    "Very true",
+                    "Extremely True",
+                ],
+                "scoring": {
+                    "not_at_all_true": 0,
+                    "a_little_true": 0,
+                    "kind_of_true": 0,
+                    "very_true": 3,
+                    "extremely_true": 5,
+                },
```

Comment:
I see we have scoring for one question here, but none of the others. Should we maybe just use the same scoring here for all of the questions?